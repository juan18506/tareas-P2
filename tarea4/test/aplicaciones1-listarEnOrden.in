# se asume un ID mÃ¡ximo de persona de 10

crearTTablaPersonas 20

# creo Persona 5 y agrego a tabla
crearAgenda
crearPersona 5 10 Persona5
crearFecha 5/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 6 y agrego a tabla
crearAgenda
crearPersona 6 10 Persona6
crearFecha 6/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 3 y agrego a tabla
crearAgenda
crearPersona 3 10 Persona3
crearFecha 9/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarPersonaEnTabla

# creo Persona 2 y agrego a tabla
crearAgenda
crearPersona 2 10 Persona2
crearFecha 5/10/2020
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# creo Persona 10 y agrego a tabla
crearAgenda
crearPersona 10 10 Persona10
crearFecha 19/10/2022
crearEvento 1 Evento1
agregarEventoPersona
insertarPersonaEnTabla

# el orden debe ser 2, 6, y 3
listarEnOrden 3 Persona6 Persona3 Persona2

liberarTTablaPersonas

Fin