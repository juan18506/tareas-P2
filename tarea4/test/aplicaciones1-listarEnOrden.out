1>#  se asume un ID máximo de persona de 10.
2>3>#  creo Persona 5 y agrego a tabla.
4>5>6>7>8>9>10>#  creo Persona 6 y agrego a tabla.
11>12>13>14>15>16>17>#  creo Persona 3 y agrego a tabla.
18>19>20>21>22>23>Persona 3: Persona3, 10 años
Evento 1:  Evento1
Fecha: 9/10/2022
24>25>#  creo Persona 2 y agrego a tabla.
26>27>28>29>30>31>32>#  creo Persona 10 y agrego a tabla.
33>34>35>36>37>38>39>#  el orden debe ser 2, 6, y 3.
40>Persona 2: Persona2, 10 años
Evento 1:  Evento1
Fecha: 5/10/2020
Persona 6: Persona6, 10 años
Evento 1:  Evento1
Fecha: 6/10/2022
Persona 3: Persona3, 10 años
Evento 1:  Evento1
Fecha: 9/10/2022
41>42>Fin.
